tests/Collection/StringCollectionTraitTest.php

```diff
@@ -4,19 +4,19 @@
 namespace MyOnlineStore\Common\Domain\Tests\Collection;
 
 use MyOnlineStore\Common\Domain\Collection\StringCollectionTrait;
+use PHPUnit\Framework\TestCase;
 
-final class StringCollectionTraitTest extends \PHPUnit\Framework\TestCase
+final class StringCollectionTraitTest extends TestCase
 {
-    public function testAsStringsWillReturnArrayOfStrings()
+    public function testAsStringsWillReturnArrayOfStrings(): void
     {
         $exception = new \Exception('bar');
 
-        /** @var StringCollectionTrait $trait */
         $trait = $this->getMockForTrait(StringCollectionTrait::class);
         $trait->expects(self::once())
             ->method('toArray')
             ->willReturn([0.00, $exception, 1337]);
 
-        self::assertEquals(['0.00', (string) $exception, '1337'], $trait->asStrings());
+        self::assertEquals(['0', (string) $exception, '1337'], $trait->asStrings());
```

Comment:
This seems wrong, why has the key been altered?

Reply:
All these alterations are made because the upgraded package seems to handle the rounding different from before (as you can see via the passing tests, even though this seems wrong).
I haven't been able to pinpoint the exact cause, because the `litipk/php-bignumbers` package is `>= 7.0`, so the same version is used as before.

Reply:
But this "rounding" has nothing to do with php-bignumbers, does this comment refer to `NumberTest`?

Reply:
> But this "rounding" has nothing to do with php-bignumbers, does this comment refer to NumberTest?

Ah yes, didn't see this single comment wasn't about that class. This particular case should have never worked: https://3v4l.org/HVKaS , no idea how the test succeeded before.

Reply:
Odd, I would expect the decimals to be included in the conversion. I guess the new PHPUnit version is more strict in array comparison